web/src/components/CityContentFooter.tsx

```diff
@@ -34,10 +34,11 @@ type CityContentFooterProps = {
 }
 
 const CityContentFooter = ({ city, language, mode = 'normal' }: CityContentFooterProps): ReactElement => {
-  const { aboutUrls, privacyUrls } = buildConfig()
+  const { aboutUrls, privacyUrls, accessibilityUrls } = buildConfig()
   const { t } = useTranslation(['layout', 'settings'])
   const aboutUrl = aboutUrls[language] || aboutUrls.default
   const privacyUrl = privacyUrls[language] || privacyUrls.default
+  const accessibilityUrl: string | undefined = accessibilityUrls?.[language] ?? accessibilityUrls?.default
```

Comment:
:wrench: Is the type actually necessary?
```suggestion
  const accessibilityUrl = accessibilityUrls?.[language] ?? accessibilityUrls?.default
```

Reply:
You're right, it's not necessary. The linter was complaining about the missing types but that must have been at a different version, it's not complaining now 🤷